README.md

```diff
@@ -30,12 +30,12 @@ It is designed primarily for web-based user interfaces for ASAB microservices bu
 TODO ...
 
 ```
-[ASAB UI App] -- [ HTTP proxy (API_URL) ] -- [ actual microservices ]
+[ASAB UI App] -- [ HTTP proxy (ASAB_MICROSERVICE) ] -- [ actual microservices ]
```

Comment:
`ASAB_MICROSERVICE` this term should not be used - it is a `SERVICE_URL` ... but look at the URL composition proposal first.